RefRed/reduction/individual_reduction_settings_handler.py

```diff
@@ -50,6 +50,29 @@ def retrieve(self):
         self._tof_range = self.get_tof_range()
         self._output_workspace_name = self.define_output_workspace_name(run_numbers=self._data_run_numbers)
 
+    def to_dict(self):
+        """
+        Return a dictionary with the reduction options
+        """
+        self.retrieve()
+        pars = dict(
```

Comment:
same comment as before, using a helper list

Reply:
I think we have to do something different here since the names aren't the same in the two objects.